src/it/scala/com.rbmhtechnology.calliope/TransactionalEventProducerSpec.scala

```diff
@@ -404,6 +405,22 @@ class TransactionalEventProducerSpec extends KafkaSpec with MustMatchers with Ty
             .expectNextN(2).map(e => e.value().payload) must contain inOrder(Event("1", "agg1"), Event("2", "agg2"))
         }
       }
+      "stop after being restarted" in {
+        val (topic, consumer) = topicConsumer()
+        val writer = runTransactionalEventProducer(topic, WritableEventStore(), settings)
+
+        whenReady(Future.sequence(Vector(
```

Comment:
we could do something like this
```
whenReady(Future.sequence(Vector(
  producer.stop(),
  akka.pattern.after(5.millis, system.scheduler) { producer.run() },
  akka.pattern.after(10.millis, system.scheduler) { producer.stop() }
))){ _ =>
```
although the messages should be sent in the invocation-order anyways event without using `after`.